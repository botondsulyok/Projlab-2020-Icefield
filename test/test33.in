field start 0
field end 0
neighbours start end
neighbours end start
entity pb icebear start
entity esk Eskimo end
item tent1 Tent esk
tent esk
stats start
stats end
stats esk
stats pb
move pb end
stats start
stats end
stats esk
stats pb