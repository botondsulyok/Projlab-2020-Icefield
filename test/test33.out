id: start
type: iceunit
building: plains0
snow: 0
pickable: null
standers: pb
pullers: null
masters: null
neighbours: end

id: end
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: start

id: esk
type: eskimo
iceUnit: end
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: pb
type: icebear
iceUnit: start
movebehaviour: amph0

Game Over
id: start
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: end

id: end
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk, pb
pullers: esk
masters: esk
neighbours: start

id: esk
type: eskimo
iceUnit: end
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: pb
type: icebear
iceUnit: end
movebehaviour: amph0

